lib/Parser/MimeDir.php

```diff
@@ -103,7 +102,7 @@ protected function parseDocument() {
 
         // BOM is ZERO WIDTH NO-BREAK SPACE (U+FEFF).
         // It's 0xEF 0xBB 0xBF in UTF-8 hex.
-        if (   3 <= strlen($line)
+        if (3 <= strlen($line)
```

Comment:
Here, maybe we should move the “separator” operator to suffix instead of prefix. I mean:
```php
if (3 <= strlen($line) &&
    ord($line[0]) === 0xef &&
    ord…)
```
?